src/index.html

```diff
@@ -5,8 +5,375 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img
+          src="/images/logo.png"
+          alt="logo"
+          class="logo__img"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class= "nav__link is-active nav__link--blue">Apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & computers
+            </a>
+
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#" class="nav__link">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <div class="cards">
+        <div class="cards__card" data-qa="card">
+          <img
+            class="cards__img"
+            src="images/imac.jpeg"
+            alt="foto"
+          >
+
+          <div class="cards__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</div>
+
+          <div class="cards__code">Product code: 195434</div>
+
+          <div class="cards__container">
+            <div class="stars stars--4">
+              <div class="star stars__star"></div>
+              <div class="star stars__star"></div>
+              <div class="star stars__star"></div>
+              <div class="star stars__star"></div>
+              <div class="star stars__star"></div>
+            </div>
+
+            <div class="cards__count">Reviews: 5</div>
```

Comment:
and here, please review all such cases in your code

Reply:
ok ,no problem